frame/support/test/pallet/Cargo.toml

```diff
@@ -13,6 +13,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"] }
+scale-info = { version = "0.10.0", default-features = false, features = ["derive"] }
```

Comment:
Does this Cargo.toml require a `scale-info/std` feature?

Reply:
It's not required but will add it anyway for consistency,